crates/astria-conductor/src/conductor.rs

```diff
@@ -193,23 +235,34 @@ impl Conductor {
                     cfg.chain_id.as_ref(),
                 ),
                 shutdown_rx,
+                sync_done_tx,
             )
             .await
             .wrap_err("failed constructing data availability reader")?;
-            data_availability_reader = Some(reader);
+            data_availability_reader = Some(da_reader);
+            shutdown_channels.insert(Self::DATA_AVAILABILITY, shutdown_tx);
```

Comment:
the task was not spawned yet - put the shutdown channel in the map when it's spawned. same for line 190